api/foreign.go

```diff
@@ -280,6 +303,55 @@ func (entity *ForeignTravel) Valid() (bool, error) {
 	return entity.List.Valid()
 }
 
+// ClearNos clears the "no" answers on application rejection
+func (entity *ForeignTravel) ClearNos() error {
+	entity.HasForeignTravelOutside.ClearNo()
+
+	// One of the few "Yes" answers that must be cleared
+	if entity.HasForeignTravelOfficial != nil && entity.HasForeignTravelOfficial.Value == "Yes" {
+		entity.HasForeignTravelOfficial.Value = ""
+	}
+
+	questionedErr := entity.List.ClearBranchItemsNo("Questioned")
```

Comment:
this method would be a great place to start with a list of keys and loop through clearing the branch items & returning any error that surfaces. :)

Reply:
I changed ClearBranchItemsNo to take multiple keys to clear

Reply:
just read that commit. I love it! :)